REFERENCE.md

```
@@ -5640,6 +5644,93 @@ await scalekitClient.auth.updateLoginUserDetails(
 
+</dd>
+</dl>
+</details>
+
+## Events
+
+<details><summary><code>client.events.<a href="https://github.com/scalekit-inc/scalekit-sdk-node/blob/main/src/events.ts">listEventsPaginated</a>(pageSize?, pageToken?, filter?) -> Promise&lt;ListEventsPaginatedResponse&gt;</code></summary>
+<dl>
+<dd>
+
+#### 📝 Description
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)